.github/actions/build-vars/action.yml

```diff
@@ -234,11 +234,18 @@ runs:
           echo "has_frontend_change=${{ steps.filter.outputs.frontend_react }}" >> $GITHUB_OUTPUT
         fi
 
-    - uses: azure/login@a65d910e8af852a8061c627c456678983e180302
+    - uses: azure/login@v2
```

Comment:
For reference, this project has traditionally pinned dependencies by hash for the reasons outlined here: https://blog.rafaelgss.dev/why-you-should-pin-actions-by-commit-hash

Reply:
Thank you for pointing that out but at some point we need to catch up with the latest release for these GitHub Actions. Nevertheless, we are just attempting to determine what if it works but not forcing this to be a change merged into the main branch yet.

Reply:
This action is from a [verified source](https://github.com/marketplace/actions/azure-login) (Azure) and has defined CODEOWNERS, so I am not that concerned with sha pinning at the moment.